go.mod

```diff
@@ -26,8 +23,8 @@ require (
 	github.com/GoogleCloudPlatform/testgrid v0.0.1-alpha.4
 	github.com/NYTimes/gziphandler v0.0.0-20170623195520-56545f4a5d46
 	github.com/andygrunwald/go-gerrit v0.0.0-20190120104749-174420ebee6c
-	github.com/aws/aws-k8s-tester v0.0.0-20190114231546-b411acf57dfe
-	github.com/aws/aws-sdk-go v1.27.4
+	github.com/aws/aws-k8s-tester v0.6.2
```

Comment:
it looks like this update is causing the build failures (cc @gyuho):
```
/b/f/w/kubetest/eks.go:141:9: dp.cfg.AWSK8sTesterPath undefined (type *eksconfig.Config has no field or method AWSK8sTesterPath)
/b/f/w/kubetest/eks.go:169:9: dp.cfg.AWSK8sTesterPath undefined (type *eksconfig.Config has no field or method AWSK8sTesterPath)
/b/f/w/kubetest/eks.go:185:9: dp.cfg.AWSK8sTesterPath undefined (type *eksconfig.Config has no field or method AWSK8sTesterPath)
/b/f/w/kubetest/eks.go:197:11: dp.cfg.ClusterState undefined (type *eksconfig.Config has no field or method ClusterState)
/b/f/w/kubetest/eks.go:199:10: dp.cfg.ClusterName undefined (type *eksconfig.Config has no field or method ClusterName)
/b/f/w/kubetest/eks.go:200:10: dp.cfg.ClusterState undefined (type *eksconfig.Config has no field or method ClusterState)
/b/f/w/kubetest/eks.go:217:15: dp.cfg.ClusterState undefined (type *eksconfig.Config has no field or method ClusterState)
/b/f/w/kubetest/eks.go:226:9: dp.cfg.AWSK8sTesterPath undefined (type *eksconfig.Config has no field or method AWSK8sTesterPath)
/b/f/w/kubetest/eks.go:243:9: dp.cfg.AWSK8sTesterPath undefined (type *eksconfig.Config has no field or method AWSK8sTesterPath)
/b/f/w/kubetest/eks.go:252:9: dp.cfg.AWSK8sTesterPath undefined (type *eksconfig.Config has no field or method AWSK8sTesterPath)
/b/f/w/kubetest/eks.go:252:9: too many errors
```

Reply:
yeah, updating anything other than controller-runtime itself and the staging dependencies wasn't intended, so I will just revert this part rather than fixing it.